src/routes/event.ts

```diff
@@ -1,16 +1,18 @@
-import { Router } from "express";
+
 import {
   getEvents,
   getUpcomingEvents,
   getPastEvents,
   createEvent
 } from "../controllers/event";
 
-const router = Router();
-
-router.get("/", getEvents);
-router.get("/upcoming", getUpcomingEvents);
-router.get("/past", getPastEvents);
-router.post("/", createEvent);
+const baseUrl = "/api/v1/";
```

Comment:
There is a lot of repetition with setting up the base url. Ideally, the base url should be set in a single file, to make maintanance easier. Suggestion would be to set base url in the `app.ts` file like this:
```
const baseUrl = "/api/v1/";
this.app.uses(baseUrl, routes);
```

then in `src/routes/index.ts`, instead of having only the test route, register all routes there (events, location etc). This way we have routes managed centrally and we avoid repetition. This means in your `app.ts`, you'll import only a single file for routes i.e `src/routes/index.ts` which handles the registration for all routes across the application.

Reply:
Cool, this can be dealt with easily using `expess`'s inbuilt `Router()` class. Thanks for this